orbs-test-keys.json

```diff
@@ -0,0 +1,52 @@
+{
+  "user1": {
```

Comment:
please give meaningful names to these users so we know what they do in these context. and remove any unused user... this invites reusing these users in the future and we should avoid it...